### ISSUE TITLE
Foldseek-multimer returns non-interacting chains

### ISSUE THREAD

**Issue by gezmi** [steineggerlab/foldseek#274]
Hi,

As searching for protein-protein interactions, my aim was to find similar interfaces in the PDB with Foldseek-multimer. I thought it would only return hits where the two similar chains are in interaction (as in the input), not just part of the same PDB. 

When I input 7o54 as the query, the results also contain e.g. 8gym, g3+M2 or G3+m2 chains as output.
Upon looking in the structure (solved by cryo-em), I discovered that these are on very different parts of the complex, there are no interfaces between them.

job: https://search.foldseek.com/result/_z6pfyCzI7Azule11oos2fHsKk5hBnJbkcjzWw/0

Is this an expected output or a bug?

Thank you for the help!

**Comment by martin-steinegger** [comment]
This looks very broken! Any idea why the query is missing @milot-mirdita @gamcil?

**Comment by gezmi** [comment]
I do not think that this is not the same issue as #271 @milot-mirdita . The query can be found and the 3D viewer is showing the matches, however I am not sure that the matches that I mentioned should even be among the hits. But I might have misunderstood how foldseek-multimer works. I expect hits with similar interfaces, but in 8gym, the chains that are returned as hits are not contacting with each other.

**Comment by milot-mirdita** [comment]
It's not the same, however, rendering was completly broken. I deployed a fix for #271 already last week that fixed rendering. I didn't have time to look at what the actual problem is.

**Comment by martin-steinegger** [comment]
Does this still persists?
